### PR TITLE
dev mode updates

### PR DESCRIPTION
**Pull request by bmicklea**
* Added dev mode to Quick Start
* Updated readme to reference dev mode and updated the "what's new section"
* Updated the roadmap to reflect the last release

**Note** this PR is dependent on completion of #307

### CODE REVIEW

**Review by javisperez — APPROVED** [review]

**Comment by javisperez** [comment]
The CI is failing

File: ROADMAP.md [hunk 1]
```diff
@@ -6,62 +6,50 @@ As a public community project the KitOps roadmap is always open to feedback. If
...
-* More CLI distribution options
-* Documentation of the KitOps workflow
+### ModelKit Chaining 🛳️
 
-## Roadmap Details
+A ModelKit can refer to another ModelKit's model as its "base model." This is useful for situations where a user wants to take an existing model (e.g., Llama 3) but create their own model based on it. The new ModelKit would include a reference to the Llama 3 ModelKit and Kit would understand to pull both when the user asks for the new ModelKit.
 
-### Dev Mode (in-progress)
+* Add chaining to ModelKit schema 
+* Add support for chaining to relevant Kit CLI commands
```
Comment (amisevsk) [inline ROADMAP.md:19]: There's trailing whitespace on these lines (causing the CI failure).

If you're using VS Code, you can trim trailing whitespace via pressing (cmd+k, cmd+x), or cmd+shift+p and searching for "trim trailing whitespace"

File: docs/src/docs/modelkit/compatibility.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
+* Clear ML
 * Comet ML
 * Databricks
 * DataRobot
 * Domino
-* Docker
-* Docker Hub
+* Docker and Docker Hub
 * DvC
+* git and git lfs
```
Comment (amisevsk) [inline docs/src/docs/modelkit/compatibility.md:18]:
```suggestion
* Git and Git LFS
```

File: docs/src/docs/quick-start.md [hunk 3]
```diff
@@ -104,15 +104,47 @@ You'll see a new entry named the same as your pack command.
...
+mkdir devmode
+cd devmode
+```
+
+Now pull and unpack an LLM ModelKit (we have [several](https://github.com/orgs/jozu-ai/packages)), I've chosen Phi3:
+
+```sh
+kit pull ghcr.io/jozu-ai/phi3:3.8b-mini-instruct-4k-q4_K_M
+kit unpack ghcr.io/jozu-ai/phi3:3.8b-mini-instruct-4k-q4_K_M
+```
```
Comment (amisevsk) [inline docs/src/docs/quick-start.md:129]: You can also unpack directly here, if we want this to be even simpler (i.e. skip pull)

**Review by amisevsk — APPROVED** [review]
:+1: with optional suggestions

**Pull request merged**